connector-email/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.redhat.cloud.notifications</groupId>
+        <artifactId>notifications-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>notifications-connector-email</artifactId>
+
+    <dependencies>
+        <!-- For template compiling and rendering -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-qute</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-scheduler</artifactId>
+        </dependency>
+        <!-- Scope: compile -->
+        <!-- Notifications module -->
+        <dependency>
+            <groupId>com.redhat.cloud.notifications</groupId>
+            <artifactId>notifications-connector-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.redhat.cloud.common</groupId>
+            <artifactId>insights-notification-schemas-java</artifactId>
+            <version>${insights-notification-schemas-java.version}</version>
+        </dependency>
+
+        <!-- Camel -->
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-http</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-bean</artifactId>
+        </dependency>
```

Comment:
In the `FETCH_AND_FILTER_RECIPIENTS` route, in the choice's first `when` statement:

``` .when(simpleF("${exchangeProperty.%s.groupUUID} == null", ExchangeProperty.CURRENT_RECIPIENT_SETTINGS))```

That requires the bean dependency, since otherwise it throws the following:

```2023-08-21 08:49:33,729 ERROR [org.apa.cam.qua.mai.CamelMainRuntime] (Quarkus Main Thread) Failed to start application: org.apache.camel.FailedToCreateRouteException: Failed to create route route1 at: >>> Split[simple{${exchangeProperty.recipient_settings}} -> [SetProperty[current_recipient_settings, simple{${body}}], Choice[[When[simple{${exchangeProperty.current_recipient_settings.groupUUID} == null} -> [To[direct://fetch-users]]]] Otherwise[[Log[Togrouphehee], To[direct://fetch-group-rbac]]]]]] <<< in route: Route(route1)[From[direct://fetch-filter-recipients] -> [OnE... because of No language could be found for: bean```